benchmarks/nemo/nemolite2d/psykal/Makefile

```diff
@@ -127,14 +126,15 @@ nemolite2d_ocl: serial_libs fortcl_lib nemolite2d_alg.f90 psyclone_scripts/ocl_t
 	cp namelist $@/.
 	cp Makefile_gen $@/Makefile
 	# Currently we need all kernels in a single OpenCL file
-	cat $@/*.cl >> $@/allkernels
-	mv $@/allkernels $@/allkernels.cl 
-	@echo "OpenCL version prepared."
-	@echo "Run 'make <target>' inside the $@ folder"
+	cat $@/*.cl >> $@/allkernels.cl
+	cd $@ && make nemolite2docl
+	@echo "OpenCL version prepared. There are additional optional targets inside"
```

Comment:
I can build this but if I then `cd nemolite2d_oce; make run_opencl_jit` then the reported checksums at the end of the run are zero. Is that expected? I'm running this on my laptop:

    Number of OpenCL Platforms:  1
    Number of OpenCL Devices:  1
    Selected Device (# 1, Compute Units:   8) -  pthread-Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
    Creating OpenCL Context for  1 devices of platform  1

Reply:
Not expected, I get the correct checksums (although with eventual segfaults from time to time). Which OpenCL implementation do you use? Make sure you made a `git submodule update`. Can you check if your `../manual_versions/psykal_opencl` and `../manual_versions/psykal_c_opencl` have the same issue? Also you can check changing the OpenCL options back to `kern.set_opencl_options({'local_size': 1})` in the ocl_transform.py script?

Reply:
Hmm, if I run the manual Fortran version then that too gives zeroes. However, if I run the manual C version then it works correctly and the OpenCL device output is different:

    Have 1 devices
    Device 0 is: pthread-Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, type=2, version=OpenCL 1.2 pocl HSTR: pthread-x86_64-pc-linux-gnu-skylake
             double precision supported
             max work group size = 4096
             max size of each work item dimension: 4096 4096
             max compute units = 8

I've made sure that the submodules are up to date. The local-sizes in both manual versions are [64,1]. Looks like it's something to do with the way FortCL interfaces to the pocl installation on my laptop? Please could you create a new Issue for this. (The `vadd` test for FortCL reports that it is successful on my laptop.)